ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js.txt

```diff
@@ -1,5 +1,6 @@
 #base=js
 
+console-ui.delete.js
```

Comment:
Do not put it before common. The object `Granite.ELC` is only initialized in common. So in this very script `Granite.ELC` could very well be null